### DISCUSSION TITLE
Add information about author that direct an article in author schema

### DISCUSSION THREAD

**Discussion by PierreSolomon** [quarto-dev/quarto-cli#3939] [Feature Requests]
Hello,

In the current author schema, it is possible to indicate if multiple author contributed jointly. In my field of work (biology), it is common to have multiple first authors (the one that write the paper and do the most important analysis) and it is possible to have multiple last authors (the ones that direct the work).

For example, if you look at the author affiliation of the [next paper](https://www.atsjournals.org/doi/10.1164/rccm.202110-2324OC?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed):

    After the name of the 4 first authors, there is the symbol "*" to indicate that they contributed equally to the work.
    After the name of the 2 last authors, there is the symbol "‡" to indicate that they jointly directed the work.

The signification of the symbols are described in the PDF:

    *These authors contributed equally to this study.
    ‡These authors jointly directed this study.

 Would it be possible to add a field in the quarto normalized schema to indicate if one or more authors direct a paper?

I previously asked this question on the github of the [author-block extension](https://github.com/kapsner/authors-block/issues/3) but I was redirected here as it isn't possible without modifying the current schema.

Thank you in advance for your answer.

**Comment by mcanouil** [comment]
The issue is that there could also be a group of second and third authors, for instance in major genetic consortium.
So if Quarto team wants to increase the "joint" author scheme, then it should probably be more general, i.e., be able to specify `equal-contributor` as a numeric for position and/or "last" for last author block.
Internally, this would require from Quarto to add the symbol when there are at least two authors with a particular value for `equal-contributor`.

**Reply by PierreSolomon** [reply]
Thank you for the answer. Would it also be possible to add a way to give a description of those groups?

**Reply by mcanouil** [reply]
Possibly, but that something for the Quarto Dev team to define/discuss/decide.

**Reply by cderv** [reply]
@dragonstyle you know the schema better than us. What is your take on this ? Should we open a feature request as issue to consider this evolution ? 

I think we encounter this in one of our journal's template also. 🤔

**Reply by dragonstyle** [reply]
I am hoping to make some improvements to the authors schema with an eye towards improving support for roles (e.g. perhaps inspired by `CRediT` (<https://www.elsevier.com/authors/policies-and-guidelines/credit-author-statement>) and improving support for group authorship in our 1.3 release. 

We'll need to look more closely at downstream support (e.g. JATS) which I will do as a part of that work. In addition, if you can point me a journals (especially if they provide LaTeX templates) that illustrate / support these scenarios that is very helpful as we try to make it straightforward to use these features within such templates...

**Reply by cderv** [reply]
I am thinking of our plos template (https://quarto-journals.github.io/plos/, https://github.com/quarto-journals/plos). 
![image](https://user-images.githubusercontent.com/6791940/212340686-b0ecb686-00e3-4347-99c3-d4c3e55224ee.png)

We had the challenge of several authors for equal contribution with symbol, which makes me think of @PierreSolomon example above. 
It was solved by a [custom templates ](https://github.com/quarto-journals/plos/blob/main/_extensions/plos/partials/_authors.tex)  and [specific attributes in YAML fields](https://github.com/quarto-journals/plos/blob/main/template.qmd#L14-L48) so not in schema. 

Not sure how it is represented in JATS...

**Comment by mcanouil** [comment]
See <https://quarto.org/docs/journals/authors.html#roles>.
